IPIP/0001-lightweight-improvement-proposal-process.md

```diff
@@ -54,29 +59,33 @@ document** based on the template in [`ipfs/specs/IPIP/0000-template.md`](./0000-
 When a new specification file is added to the repo, it should be based on
 the template at [`ipfs/specs/template.md`](../template.md).
 
-### Reviewing IPIPs
-
-[Specs Stewards] will review new IPIP PRs during periodical (best-effort) triage.
+When naming a new proposal, don't try to introduce an IPIP number; we will do that only for
+IPIPs that are approved before accepting into master.
 
-IPFS Community is encouraged to participate in the review process.
+Proposals are officially submitted when a pull request into master is opened
 
-IPIP can be either:
-- merged,
-- rejected (PR close without merging),
-- deferred (converting PR back to a draft).
+Proposals that were reviewed and rejected will be moved into /archive folder and then merged into master
```

Comment:
I think this should be the same place as one mentioned in "Reviewing IPIPs" section, so:

```suggestion
Proposals that were reviewed and rejected will be moved into `/IPIP/rejected` folder and then merged into `main`.
```

nit: this will be a lossy process, because IPIP PR may have changes to existing specs outside the `/IPIP` dir.
I guess we could link the original PR at the top of an archived IPIP document, in case someone wants to see the full diff IPIP proposed?